dev/bin/knative-installer.sh

```diff
@@ -15,26 +15,21 @@ else
   reset=''
 fi
 
-CONFIGURE_SCRIPT_DIR_PATH=`dirname "${BASH_SOURCE[0]}"`
-# The Knative Core APIs, like Broker or Trigger from midstream
-eventing_core_url=${CONFIGURE_SCRIPT_DIR_PATH}/knative/resources/knative-eventing.yaml
-# Knative Kafka offering: All Kafka centric APIs from midstream
-eventing_kafka_url=${CONFIGURE_SCRIPT_DIR_PATH}/knative/resources/knative-kafka.yaml
-
 function header_text {
   echo "$header$*$reset"
 }
 
 header_text "Knative Eventing Kafka - Installer"
 
 header_text "Initializing Knative Eventing Core APIs"
-kubectl apply --filename $eventing_core_url
+kubectl apply -f https://raw.githubusercontent.com/openshift/knative-eventing/release-v1.4/openshift/release/artifacts/eventing-core.yaml
+kubectl create -f https://raw.githubusercontent.com/openshift/knative-eventing/release-v1.4/openshift/release/artifacts/eventing-post-install.yaml
 
 header_text "Waiting for Knative Eventing Core to become ready"
 kubectl wait deployment --all --timeout=900s --for=condition=Available -n knative-eventing
 
 header_text "Initializing Knative Eventing Kafka APIs"
-kubectl apply --filename $eventing_kafka_url
+kubectl apply -f https://raw.githubusercontent.com/openshift-knative/eventing-kafka-broker/release-v1.4/openshift/release/artifacts/eventing-kafka.yaml
```

Comment:
@r00ta It looks like the `securityContext` definitions in this file are preventing the pods from booting on vanilla k8s. In particular the `runAsNonRoot:true` causes it to fail on my local.

Going to the upstream https://github.com/knative-sandbox/eventing-kafka-broker/releases/download/knative-v1.4.3/eventing-kafka.yaml for 1.4.3 of eventing-kafka, pods all boot as expected